webapi/src/main/scala/org/knora/webapi/responders/IriService.scala

```diff
@@ -6,15 +6,12 @@
 package org.knora.webapi.responders
 
 import com.typesafe.scalalogging.LazyLogging
-import zio.ZIO
-import zio._
-
 import dsp.errors._
 import org.knora.webapi.IRI
-import org.knora.webapi.messages.SmartIri
-import org.knora.webapi.messages.StringFormatter
+import org.knora.webapi.messages.{SmartIri, StringFormatter}
 import org.knora.webapi.messages.StringFormatter.MAX_IRI_ATTEMPTS
 import org.knora.webapi.store.triplestore.api.TriplestoreService
+import zio.{ZIO, _}
```

Comment:
does that import make sense? if `_` is imported, then `ZIO` should be redundant, right?

Reply:
yes, it is redundant.
Sigh, imports.... We should fix scalafix to take care of them for us, I am not inclined to maintain these manually and to discuss them on every pull request.
=> DEV-1959